sympy/physics/mechanics/body.py

```diff
@@ -0,0 +1,167 @@
+from sympy import Symbol
+from sympy.physics.mechanics import RigidBody, Particle, ReferenceFrame, \
+    inertia
+from sympy.physics.vector import Point, Vector
+
+__all__ = ['Body']
+
+
+class Body(RigidBody, Particle):
+    """
+    A Body which can be connected by joints.
+
+    Instances of Body can be connected using specific joints and can further
+    be used to generate equations of motion. It can be seen as an extension
+    of already present RigidBody and Particle classes and can inherit from
+    either of them based on the arguments passed.
+
+    Parameters
+    ---------
+    name: string
+        defines the name of the body. It is used as the base for defining body
+        specific properties.
+    masscenter : Point (optional)
+        The point which represents the center of mass of the rigid body.
+    frame : ReferenceFrame (optional)
+        The ReferenceFrame which the rigid body is fixed in.
+    mass : Sympifyable (optional)
+        The body's mass.
+    body_inertia : Dyadic (instance of inertia)
+        The body's inertia about center of mass.
+
+    Example:
+    --------
```

Comment:
Remove the `:` and it should be `Examples` I believe. See: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt